### DISCUSSION TITLE
gitea support

### DISCUSSION THREAD

**Discussion by NejatHakan** [giscus/giscus#1580] [Q&A]
Hi, does anybody know if gitea is supported? I could not find any information about it.
I would rather like to use my self hosted gitea server instead of github.
If there is actually no support for gitea, is it at least planned or even better allready in progress?

Best regards and thank you in advance

**Comment by laymonage** [comment]
Does Gitea have a Discussions feature?

**Reply by niclasheinz** [reply]
I think it has no discussion functionality. However, we could work with issues, and create issues with the label discus for example. What do you think @laymonage ?

**Reply by CheeksTheGeek** [reply]
I think the prime focus of giscus is utilizing GitHub Discussions, while there's other systems that focus on GitHub Issues, like [Utteranc.es](https://utteranc.es/) and [gitalk](https://gitalk.github.io/). I can't speak for @laymonage but from looking at the directions given for migration from those other tools [here](https://giscus.app/#:~:text=component%20library.-,Migrating,-If%20you%27ve%20previously), it seems like this kind of a request to accomodate Gitea's Issues would be better for those tools.

**Reply by milahu** [reply]
a more generic alternative would be [hypothesis annotations](https://github.com/hypothesis/client) of HTML files
identified by their [canonical URL](https://github.com/hypothesis/client/blob/bf7b6d507e8cccea2d94f870b10c06ebc2fa5b76/src/annotator/integrations/html-metadata.ts#L60)
